lms/static/sass/views/_login-register.scss

```diff
@@ -371,6 +369,7 @@ $sm-btn-linkedin: #0077b5;
             height: 34px;
```

Comment:
Is this needed (especially with your definition at https://github.com/edx/edx-platform/pull/8603/files#diff-31cc02703be2a91e088f5d3ebe1a4360R355)? Can this just be the height of that parent?